doc/man3/EVP_MAC.pod

```diff
@@ -80,6 +80,45 @@ rely on an underlying computation algorithm.
 
 EVP_MAC_fetch() fetches an implementation of a MAC I<algorithm>, given
 a library context I<libctx> and a set of I<properties>.
+
+The OpenSSL default provider supports the following algorithms:
```

Comment:
This adds a duplicate of the lists that are already in `doc/man7/OSSL_PROVIDER-*.pod`.  I would rather just have a reference, like this:

```
See L<OSSL_PROVIDER-default.pod/Message Authentication Code (MAC)> for the lists of
algorithms supported by the default provider.
```